Sources/Datadog/RUM/Integrations/CrashReportReceiver.swift

```diff
@@ -399,7 +399,7 @@ internal struct CrashReportReceiver: FeatureMessageReceiver {
                 domComplete: original.view.domComplete,
                 domContentLoaded: original.view.domContentLoaded,
                 domInteractive: original.view.domInteractive,
-                error: original.view.error,
+                error: .init(count: original.view.error.count + 1),
```

Comment:
Error model for now only has count, but I think it would be safer to copy object `original.view.error` and then add 1 to the count (just so it doesn't break when we alter the model).